From fd73bb7cf812fcb96116f70393d3e66aa4ddc1fa Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <vladimir.oltean@nxp.com>
Date: Fri, 6 Dec 2019 13:56:46 +0200
Subject: [PATCH] arm64: dts: ls1028a: Disable swp5 by default

This was missed when moving the CPU port and disabling eno3.

Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -810,6 +810,7 @@
 					port@5 {
 						reg = <5>;
 						phy-mode = "gmii";
+						status = "disabled";
 
 						fixed-link {
 							speed = <1000>;
